src/components/AdminPanelLayout/UsersPane.tsx

```diff
@@ -143,12 +148,37 @@ const UsersPane = (props: Props) => {
     }
   };
 
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.currentTarget.value) {
+      const foundUsers: User[] = [];
+      users.map((u: User) => {
+        const regex = new RegExp(`${u.firstName}|${u.lastName}|${u.email}|${u.role}`, 'ig');
+        if (event.currentTarget.value.search(regex) >= 0) {
+          foundUsers.push(u);
+        }
+      });
+      setUsersToDisplay(foundUsers);
+    }
+    else {
+      setUsersToDisplay(users);
+    }
```

Comment:
```suggestion
    const searchValue = event.currentTarget.value.toLowerCase().trim();
    const foundUsers = searchValue ? users.filter((u: User) => {
      const searchString = `${u.firstName}|${u.lastName}|${u.email}|${u.role}`.toLowerCase();
      return searchString.indexOf(searchValue) !== -1;
    }) : users;
    setUsersToDisplay(foundUsers);
```

Reply:
@jasekiw pushed a commit for this, made a small adjustment to the search string so it would also match a <firstname><space><lastname> search.